### ISSUE TITLE
[Package Issue]: qBittorrent.qBittorrent

### ISSUE THREAD

**Issue by sirinath** [microsoft/winget-pkgs#95864]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Other

### Brief description of your issue

qBittorrent has many variation like Qt6 lt2, Qt5 lt2, Qt6, Qt5, etc.

![image](https://user-images.githubusercontent.com/637415/217447961-8a4e4175-d5a9-4013-8bdf-4d55e2fdc4e3.png)

This does not have the option to select the variation.

### Steps to reproduce

N/A

### Actual behavior

No option to choose the desired build.

### Expected behavior

Should be able to choose the desired build.

### Environment

```shell
NA
```

**Comment by nklowns** [comment]
Hello, it would be great to see this solved.

**Comment by SpecterShell** [comment]
WinGet can not distinguish which variation of qBittorrent has been installed because all of them write the same ARP information to the registry, with which WinGet detects the programs installed as well as their versions. The variation with the latest Qt and libtorrent, which should have most features, is chosen as they did not suggest the recommended variation anywhere.
![image](https://github.com/microsoft/winget-pkgs/assets/56779163/1e0d4bf6-0ad9-497e-9255-2e7060e5fc1a)

**Comment by sirinath** [comment]
@SpecterShell 

As suggested the best choice is:
- Latest QT
- Latest libtorrent

**Comment by SpecterShell** [comment]
Yeah currently the variation with libtorrent 2.0 (lt20) and Qt 6 is selected, both of which are the latest.

**Comment by masterflitzer** [comment]
this makes the existence of `qBittorrent.qBittorrent.Qt6` in addition to `qBittorrent.qBittorrent` a little confusing, neither of the package names indicate which version is which, one has to dig into the repo to find out...

**Comment by SpecterShell** [comment]
> this makes the existence of `qBittorrent.qBittorrent.Qt6` in addition to `qBittorrent.qBittorrent` a little confusing, neither of the package names indicate which version is which, one has to dig into the repo to find out...

`qBittorrent.qBittorrent` has been moved to the libtorrent 1.0 edition, and `qBittorrent.qBittorrent.Qt6` should be the libtorrent 2.0 edition, proposed by #177048.

There used to be 4 editions (Qt5/6 x libtorrent 1/2), but since 5.0 there are only two remaining and both of them seem to be based on Qt6...

**Comment by masterflitzer** [comment]
thanks for the clarification, wouldn't it make sense to rename them to `qBittorrent.qBittorrent.lt1` and `qBittorrent.qBittorrent.lt2`?

**Comment by masterflitzer** [comment]
is this fixed by the completion of #263941 or is anything else needed?
